demo/src/main/res/values/styles.xml

```diff
@@ -8,4 +8,32 @@
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <style name="bottom_sheet_item">
```

Comment:
Very good that you defined these here, but I don't think they ever really got used :'(

Reply:
That's because I didn't take the time to do a well-defined schema for the bottom sheet, it's kind of a draft, I'm still checking it out.